Policies/Kubernetes/allowed-external-ips/azurepolicy.json

```diff
@@ -0,0 +1,65 @@
+{
+  "properties": {
+    "displayName": "Ensure only allowed External IPs are used in Kubernetes Cluster",
+    "policyType": "Custom",
+    "mode": "Microsoft.Kubernetes.Data",
+    "description": "This policy ensures only allowed external ips are used in a Kubernetes cluster. For instructions on using this policy, please visit https://aka.ms/kubepolicydoc",
+    "metadata": {
+      "version": "1.0.0-preview",
+      "category": "Kubernetes PSP",
```

Comment:
this is not a PSP

Reply:
Good catch. It is a service, not pod. Fixed.